yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -353,7 +353,7 @@ private[yarn] class YarnAllocator(
 
     } else if (missing < 0) {
       val numToCancel = math.min(numPendingAllocate, -missing)
-      logInfo(s"Canceling requests for $numToCancel executor containers")
+      logInfo(s"Canceled requests for $numToCancel executor container(s)")
```

Comment:
The first message implies "I need to cancel 5". The second message says "I'm canceling 5". There is no third message that says "I cancelled 5". I don't think the second message is wrong, yes. But you could try to add that third message where appropriate. Or expand the first message to be clearer about the before and after values.